packages/logger/src/formats.ts

```diff
@@ -0,0 +1,130 @@
+import { TransformableInfo } from 'logform';
```

Comment:
Use single filename please (format.ts). See [development guidelines](https://coda.io/d/Development-Guidelines_d7DszrlIOzU/Naming_suckZ#_luLAo) for more information

Reply:
@knor-el-snor sounds good! I think we should update the docs since we probably do allow some files in plural form like `constants.ts`

Reply:
@gorjuspixels Yeah we should have an exception list